client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -107,14 +111,18 @@ def prediction_url(self, value):
             raise ValueError("not a valid prediction_url")
         self._prediction_url = parsed.geturl()
 
+    def batch_prediction_url(self):
+        # Adding the v1/ to make sure we don't accidentally replace the wrong part of the URL
+        return self.prediction_url.replace("v1/predict", "v1/batch-predict")
 
     # TODO: Implement dynamic compression via separate utility and call it from here
     def _predict(
             self,
             x: Any,
-            compress: bool=False,
-            prediction_id: Optional[str]=None,
-            ):
+            prediction_url: bytes,
```

Comment:
Is this not a `str`? 🤔

Reply:
it's... not

Reply:
idk WHY, but pycharm is very insistent that it's not a str 😒

Reply:
Hrm...Could we leave out the type hint altogether, then? There's no way it's supposed to be `bytes`.